jekyll/_cci2/local-cli.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: classic-docs
+title: "The CircleCI Local Command Line Interface (CLI)"
+short-title: "Using the CircleCI Local CLI"
+description: "How to run local jobs with the CLI"
+categories: [troubleshooting]
+order: 10
+alias: /docs/2.0/local-jobs/
+---
+
+*[Build Environments]({{ site.baseurl }}/2.0/build/) > {{ page.title }}*
+
+* TOC
+{:toc}
+
+## Overview
+
+The `circleci` commands enable you to reproduce the CircleCI environment locally and run jobs as if they were running on the hosted application for more efficient debugging and configuration in the initial setup phase. 
+
+You can also run `circleci` commands in your `config.yml` file for jobs that use the primary container image. This is particularly useful for globbing or splitting tests among containers. 
+
+**Note:** The Local CLI does not yet support Workflows.
```

Comment:
Potentially dumb question: is the phrase "local CLI" redundant? Are command-line interfaces local by definition?

Reply:
Potentially. The idea here is that there are other CLIs that we have. This one was specifically made for local CI builds.